README.md
# Denver_Northwest
facilitates the exploration of the the Highlands neighborhood and surrounding areas through data visualization and animation-heavy story-telling. Users are taken on a 'journey' through time and place to different historical landmarks and areas of import and are introduced to the constantly changing demographics of this dynamic area in Colorado's Capital


public/history
By Laura Ruttum Senturia, Library Director at the Stephen H. Hart Library & Research Center, History Colorado

The real estate business has a famous saying about what makes a particular property valuable: “location, location, location.” Location has played a large role in the development of the Highlands neighborhood over the past one hundred and fifty years, as this Denver community has seen its fortunes ebb and flow, often based on its proximity to the Platte River and the neighborhood’s shifting edges.

When discussing the Highlands, defining the neighborhood boundaries becomes important, as the outline of the area has changed significantly over the past century and a half, and there have actually been two towns in the area that went by the same name: Highland (ca. 1860s) and Highlands (1875 to 1896). In addition, historically there have been multiple other neighborhoods that are often lumped together as “the Highlands,” including Highland Park, Scottish Highlands, West Highland, Potter Highlands, and occasionally Berkeley and Sunnyside are thrown in, for good measure. The northern boundary of the neighborhood has moved from West 26th to West 38th, and the western boundary from Zuni out to Federal. To further confuse things, in the 1940s, an area of the neighborhood that previously housed businesses and people was demolished to make way for the “Valley Highway,” also known as I-25. This division essentially split Highland in half, leaving the area around Platte and Water Streets on one side of the highway, isolated from the rest of the original neighborhood to the west. The division remains today, leaving many Denverites surprised to discover that Highland technically includes the district featuring REI, My Brother’s Bar, and the collection of other shops, restaurants, and housing between I-25 and the Platte River.

Detail of a diorama depicting Denver as it looked in 1860. This diorama can be seen for free in the lobby of the History Colorado Center. Image courtesy Stephen H. Hart Library & Research Center, History Colorado
Detail of a diorama depicting Denver as it looked in 1860. This diorama, which was created as a WPA project, can be seen for free in the lobby of the History Colorado Center. Image courtesy Stephen H. Hart Library & Research Center, History Colorado
In 1858, William Larimer and David Collier platted a town on the high cliffs above where the South Platte and Cherry Creek converge, just across the river from the fledgling towns of Denver City and Auraria. They formed the Highland Town Company the following year. Larimer assumed that the area’s proximity to the water and both the Overland and Prospect Trails (today’s 15th and 38th Streets, respectively) would be helpful for trade. In actuality the river served more as a physical barrier isolating the Highland area from the business prospects of the rapidly growing twin cities below. The town of Highland never really took off as hoped, despite establishment of a ferry service across the Platte in 1859, and the first bridge, built in 1860. In April 1860, the separate town of Highland joined with Auraria and Denver City to form the new town of Denver, and the original Highland area became known simply as “North Denver.”

Despite the transportation challenges, over time people continued to settle in the area to the north and west of Denver, and in 1875, the City of Highlands incorporated just to the north of the original town site. What is now Platte Street served as one of the industrial and commercial districts for this city, with breweries, a power plant, and railroad-associated industries eventually taking hold. Initially, wealthy citizens were drawn to the residential parts of the neighborhood higher up the hill, prizing the Highlands for the views, artesian well water, and clean air up above the industrial pollution of Denver proper.

Portrait of General William Larimer painted in 1842. Image courtesy Stephen H. Hart Library & Research Center, History Colorado.
Portrait of General William Larimer painted in 1842. Image courtesy Stephen H. Hart Library & Research Center, History Colorado.
During this time period the southwest corner of current-day Highland was also laid out, built not organically but rather as a planned development. William A Bell and William Jackson Palmer planned this 288-acre plot they named “Highland Park,” intending for it to resemble a Scottish village. Many of the streets in the Highland Park section have retained their names, and so a glance at a modern map will show streets such as “Argyle,” “Dunkeld,” and “Caithness.”

Returning to earlier years, the town of Highlands had some fairly specific laws, including, from the 1889 list of ordinances, restrictions on hitching horses to trees (which were scarce and valuable, to be fair), riding horses “immoderately” (we assume they mean “no speeding!”), use of vulgar language, and kite flying. Any rowdies and ruffians with plans for profligate kite flying were surely sorely disappointed. The town also permitted liquor sales, but stipulated the cost of licenses at between $3000 and $5000 annually (at a time when that likely represented more than a decade’s worth of profits). Essentially, Highlands had the most annoying homeowners association, ever.

Much as the Five Points neighborhood saw during the same time period, when Capitol Hill became the trendy new ‘hood for the wealthy and well-connected in the 1890s, the Highlands also lost many of their rich citizens to the new district. This population loss, combined with the nationwide depression set off by the silver crash of 1893, further cemented things: the Highlands began to struggle economically. In 1896, threats from Denver’s mayor to cut bridge access between the two cities, paired with debt and a desire for better police and fire protection, led Highlands to join the list of esteemed neighboring cities Denver annexed and turned into official neighborhoods.


seeds/01_neighborhoods.js
'use strict';
exports.seed = function(knex, Promise) {
  // Deletes ALL existing entries
  return knex('neighborhoods').del()
    .then(function () {
      return Promise.all([
      // Inserts seed entries
      knex('neighborhoods').insert(
        {
          id: 1,
          name: 'Berkeley',
          description: 'Berkeley was advertised as a healthful suburb, free from the smoke and dust of the city, with plentiful water and convenient public transportation. In 1892, the mostly middle and working class residents of the community incorporated the town of North Denver, encompassing the acreage of Walker\'s farm. Its boundaries were roughly 52nd Avenue to the north, Sheridan Boulevard to the west, 44th Avenue to south, and Federal Boulevard to the east. In 1898, the town\'s name was changed to Berkeley. Commercial development was limited to small neighborhood stores and a small commercial district along Tennyson Street where the post office was established. Berkeley grew slowly and by the time it was annexed on 1 December 1902, there were only 707 residents in the town. The eastern stretches of Berkeley were especially slow to develop. In 1910 there were only three houses between Regis College and the Henry Weirich celery farm at the east end of Rocky Mountain Lake. At this time, the area was largely occupied by "truck farmers" -- farmers who took their fruits and vegetables into Denver to sell -- and the area remained this way through the 1940\'s. The Berkeley area eventually became split into two parts: Lower Berkeley and Berkeley Hill. The school for lower Berkeley was the Louisa M. Alcott Elementary School, built in 1892. The Berkeley Hill children went to Berkeley School at West 50th Avenue and Lowell. The Berkeley neighborhood continued to develop throughout the mid 20th century, largely though subdivisions. The lion\'s share of the growth occurred in the 1950s and 1960s.',
          photo_url: ''
        }),
      knex('neighborhoods').insert(
        {
          id: 2,
          name: 'Sunnyside',
          description: 'This neighborhood is considered by many to be the newest of the North Denver neighborhoods to begin gentrification. Before being called Sunnyside, the area was considered to be apart of Highlands. The neighborhood was home to poorer irish, italian and hispanic immigrants who wanted cheap housing, access to the farmland and river, and proximity to the Argo Smelter, just north of town-a major employer during the Gold Rush of the late 1800s and the railroad-also a major employer. Sunnyside, with a border on the now Lower-Highlands area, shared many of the same demographics of its southern neighbor. The area borering the two neighborhoods was the Little Italy of its day, home to two Italian churches, an orphanage and school overseen by Mother Francis Xavier Cabrini, and four Italian language newspapers, in addition to the families and businesses.  Sunnyside saw crime and neglect through the 70s, 80s, 90s and 2000s, and was considered to be the seediest area of Northwest Denver. Today, housing prices are still considerably cheaper than it\'s pricier neighbors to the south and west. With a large hispanic population, the area is quieter, more residential and borders up on warehouse disctricts and forgotten neighborhoods of Globeville and Elyria-Swansea.',
          photo_url: ''
        }),
      knex('neighborhoods').insert(
        {
          id: 3,
          name: 'West Highland',
          description: 'Just west of lo-hi, it\'s beginnings are similar to that of it\'s eastern neighbor, people began to move up the hill more readily after the flood of 1864, and that brought the expansion of the highlands area into what we now refer to as West Highland. When streetcars were introduced in______, it made access to West Highland even easier. Business centers began to pop up around street car stops, including West Highland\'s shopping and business district, \'Highland Square\'. The neighborhood is famous for the establishment of the botanical garden, zoo and amusement park, \'Elitch Gardens\'. Today, home prices are continuing to rise steadily (The average price for a home in 2000 was 100,000, and is now closer to 500,000). In 2016, West Highland was ranked 8th for best neighborhood in the nation.',
          photo_url: ''
        }),
      knex('neighborhoods').insert(
        {
          id: 4,
          name: 'Lower Highland',
          description: 'Perhaps what most people associate Northwest Denver to the most, Lower Highlands, or simply \'Lo-Hi\', has seen explosive growth over the past decade. Its proximity to downtown and plethora of restaurants make it a destination for young working professionals. This playground for Denver\'s young elite, started as a one of the first suburbs of the fledgling city. The area is also perhaps the most historic of North Denver, as Denver began its expansion in all directions in the mid 1800s. Lo-Hi was the first bit of land to be established west of downtown. In the 1850s, The Rocky Mountain News noted: \'No more handsome location for residences can be found than on the highlands of Highland, on the opposite side of the river from and overlooking Auraria and Denver, and a vast extent of surrounding territory.\'. The building of bridges, streetcars and the flood of 1864, which killed 20 people and wiped away many homes in Denver, were all factors in Highland getting it\'s start. The area was advertised as having the best water (after the discovery of a local artesian well), the cleanest air (as it sat above the rest of smoggy Denver and regularly got breezes from the west), and was considered to be above Denver, both literally and figuratively. Strict laws and guidelines were put into effect in 1889, to separate the neigbhorhood from wild ways of Denver, including: restrictions on hitching horses to trees, riding horses “immoderately” (speeding?), use of vulgar language, and kite flying. The town permitted liquor sales, but charged between $3000 and $5000 annually (at a time when that likely represented more than a decade’s worth of profits). In alot of ways, Highland was perhaps the first suburb with an HOA. All in all, It was a place to escape the undignified, \'rough and tumble\' ways of Denver. Men were expected to work in Denver, and come home to their clean and quiet home, up the hill. In the 1880 and 1890s, a new wave of neighbors began moving into the area, starting with the Irish, Italians and later Hispanics from Southwestern Colorado and Mexico primarily.',
          photo_url: ''
        }),
      knex('neighborhoods').insert(
        {
          id: 5,
          name: 'Sloan Lake',
          description: 'A mostly residential neighborhood, known for it\'s namesake-the 200-acre sized lake. The area is seeing much development just as the rest of North Denver continues to change.',
          photo_url: ''
        }),
      knex('neighborhoods').insert(
        {
          id: 6,
          name: 'Jefferson Park',
          description: 'An area just to the west of Highland, Jefferson Park was once considered a part of Highland, housing the original highland Town Hall. Today, Jefferson Park is one of the fastest in the area to change. The stream of new developments never seems to end.',
          photo_url: ''
        })
        //add all neighborhoods seed????
      ]);
    })
    .then(() => {
        return knex.raw(
            "SELECT setval('neighborhoods_id_seq', (SELECT MAX(id) FROM neighborhoods));"
        );
    });
  };
